### PR TITLE
docs: release notes for 0.13.6.

### PR DESCRIPTION
**Pull request by determined-dsw**
## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234

### CODE REVIEW

File: docs/release-notes.txt [hunk 1]
```diff
@@ -8,6 +8,35 @@
...
 **************
 
+Version 0.13.6
+==============
+
+**Release Date:** October 14, 2020
+
+**Improvements**
+
+- Agent: The ``boot_disk_source_image`` field for GCP dynamic agents and ``image_id`` field for AWS dynamic agents are now optional. They default to the latest agent images.
```
Comment (armandmcqueen) [inline docs/release-notes.txt:18]:
```suggestion
- Agent: The ``boot_disk_source_image`` field for GCP dynamic agents and ``image_id`` field for AWS dynamic agents are now optional.
```
I don't think that line is necessary and it is a little confusing (it uses a hardcoded list of images and if we create a new agent image, that new image will not be used)

File: docs/release-notes.txt [hunk 2]
```diff
@@ -8,6 +8,35 @@
...
 **************
 
+Version 0.13.6
+==============
+
+**Release Date:** October 14, 2020
+
+**Improvements**
+
+- Agent: The ``boot_disk_source_image`` field for GCP dynamic agents and ``image_id`` field for AWS dynamic agents are now optional.
```
Comment (neilconway) [inline docs/release-notes.txt:18]: What is the behavior if this field is omitted? @shiyuann

Comment (determined-dsw) [reply docs/release-notes.txt:18]: The previous commit explained it: "They default to the latest agent images.".  Should we put this wording back in @neilconway @armandmcqueen ?

Comment (neilconway) [reply docs/release-notes.txt:18]: Yes I think being more explicit here is useful.

Comment (determined-dsw) [reply docs/release-notes.txt:18]: sgtm

File: docs/release-notes.txt [hunk 3]
```diff
@@ -8,6 +8,35 @@
...
+
+- Harness: Fix a bug which affected ``TFKerasTrial`` using TensorFlow 2 with ``gradient_aggregation`` > 1.
+
+- Master: Fix a bug where the master instance would fail if an experiment could not be read from the database.
+
+- WebUI: Preserve the colors used for multiple metrics on the metric chart.
+
+- WebUI: Fix the ability to cancel a batch of experiments.
+
+- WebUI: Fix the ``humanReadableFloat`` bug on the Experiment Details page.
```
Comment (neilconway) [inline docs/release-notes.txt:38]: We should generally avoid writing release note entries that refer to internal function names or similar internal concepts. Can you suggest some revised text, @hkang1 ?

Comment (hkang1) [reply docs/release-notes.txt:38]: @neilconway Totally agree with avoiding internal functions. In this case though, the Experiment Details page failed to render and it's just a blank page, and the only distinguishing property is that there is a console error that complains about `humanReadableFloat`... Is this better? `- WebUI: Fix the issue of Experiment Details page not rendering due to a ``humanReadableFloat`` error.`

Comment (neilconway) [reply docs/release-notes.txt:38]: In what situation does the error arise?

Comment (hkang1) [reply docs/release-notes.txt:38]: ah that's a great way to look at it. How about this:
```
- WebUI: Fix the issue of the Experiment Details page not rendering when the latest validation metric is not available.
```

Comment (neilconway) [reply docs/release-notes.txt:38]: Looks great! Thanks @hkang1

**Review by neilconway — APPROVED** [review]

**Pull request merged**